packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/mentionModule.tests.js

```diff
@@ -161,32 +163,55 @@ QUnit.module('Mention format', () => {
         };
 
         const element = MentionFormat.create(data);
-        assert.strictEqual(element.innerText, '#John Smith', 'correct inner text');
+
+        new MentionFormat({}, element);
+        assert.strictEqual(element.children[0].innerText, '#John Smith', 'correct inner text');
```

Comment:
[copypaste]
I believe we can just check that `element.attr('data-marker')` is correct.
We don't need to call constructor here because its work is tested in other place.